app/javascript/dashboard/components/layout/sidebarComponents/AccountContext.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <div v-if="showShowCurrentAccountContext" class="account-context--group">
-    {{ $t('SIDEBAR.CURRENTLY_VIEWING_ACCOUNT') }}
-    <p class="account-context--name text-ellipsis">
-      {{ account.name }}
-    </p>
+    <a @click="$emit('toggle-accounts')">
```

Comment:
For A11y, it is better to use the button instead of the A tag here. A tag without an href is not advisable. I can make the change myself once @nithindavid shares the feedback on the design @jordan-brough